hail/src/main/scala/is/hail/expr/ir/StringTableReader.scala

```diff
@@ -44,11 +121,24 @@ class StringTableReader(
 
   override def pathsUsed: Seq[String] = params.files
 
-  override def lower(ctx: ExecuteContext, requestedType: TableType): TableStage =
-    executeGeneric(ctx).toTableStage(ctx, requestedType)
```

Comment:
You should also delete `executeGeneric`, now that you've deleted the place that calls it.

Reply:
🎉 🎉 🎉 🎉 🎉 🎉 🎉